src/DotNetCore.CAP/CAP.Options.cs

```diff
@@ -34,6 +34,16 @@ public CapOptions()
         /// </summary>
         public string DefaultGroup { get; set; }
```

Comment:
As described in the comments, rename `DefaultGroup` to `GroupName`  seemed more appropriate